### PR TITLE
Update sdk

### PR DESCRIPTION
**Pull request by voscob**
### Description

I've updated the `sdk` version.

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit-connector-oracle/pulls) for the same update/change.
- [ ] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: coltypes/coltypes.go [hunk 1]
```diff
@@ -52,24 +54,30 @@ func TransformRow(row map[string]any, columnTypes map[string]ColumnData) (map[st
 			continue
 		}
 
-		switch data := columnTypes[key]; {
-		case data.Type == numberType:
+		data := columnTypes[key]
+		if data.Type == numberType {
```
Comment (maksenius) [inline coltypes/coltypes.go:58]: maybe change data.Type to columnTypes[key].Type

File: coltypes/coltypes.go [hunk 2]
```diff
@@ -52,24 +54,30 @@ func TransformRow(row map[string]any, columnTypes map[string]ColumnData) (map[st
...
+			if err != nil {
+				return nil, fmt.Errorf("convert oracle number type to string: %w", err)
+			}
+
+			value, err = strconv.Atoi(valStr.(string))
+			if err != nil {
+				return nil, fmt.Errorf("convert oracle number type to int: %w", err)
+			}
+
 			if data.Precision != nil && *data.Precision == 1 && data.Scale != nil && *data.Scale == 0 {
```
Comment (maksenius) [inline coltypes/coltypes.go:69]: please add comment to explain it

File: destination/destination_integration_test.go [hunk 3]
```diff
@@ -82,20 +82,41 @@ func TestDestination_WriteIntegration(t *testing.T) {
...
+						"attr_0": "string",
+						"attr_1": 1,
+						"attr_2": false,
+					},
+					"achievements": []string{"achievement_0", "achievement_1"},
+				}},
 			},
-		})
+			{
+				Operation: sdk.OperationCreate,
```
Comment (maksenius) [inline destination/destination_integration_test.go:102]: you can change sdk.OperationCreate => to sdk.OperationSnapshot

File: destination/destination_test.go [hunk 4]
```diff
@@ -93,29 +94,49 @@ func TestDestination_Write(t *testing.T) {
...
-				"name": "Void",
+			Payload: sdk.Change{
+				After: sdk.StructuredData{
+					"id":   1,
+					"name": "John",
+				},
+			},
+		}
+		records[1] = sdk.Record{
+			Position: sdk.Position("1.0"),
```
Comment (maksenius) [inline destination/destination_test.go:115]: sdk.Position("1.0")=> please change to your position

File: destination/destination_test.go [hunk 5]
```diff
@@ -93,29 +94,49 @@ func TestDestination_Write(t *testing.T) {
 		ctrl := gomock.NewController(t)
 		ctx := context.Background()
 
-		record := sdk.Record{
+		metadata := sdk.Metadata{}
+		metadata.SetCreatedAt(time.Now())
+
+		records := make([]sdk.Record, 2)
+		records[0] = sdk.Record{
 			Position: sdk.Position("1.0"),
```
Comment (maksenius) [inline destination/destination_test.go:102]: sdk.Position("1.0") please change to Oracle position or set nil. It is not needed for destination

Comment (voscob) [reply destination/destination_test.go]: Deleted.

File: destination/destination_test.go (hunk 4 above)
Comment (voscob) [reply destination/destination_test.go]: Changed.

File: coltypes/coltypes.go (hunk 2 above)
Comment (voscob) [reply coltypes/coltypes.go]: Good point.
Added.

File: coltypes/coltypes.go (hunk 1 above)
Comment (voscob) [reply coltypes/coltypes.go]: Done.

**Review by maksenius — APPROVED** [review]

**Review by maksenius — APPROVED** [review]

**Pull request merged**